packages/components/src/components/toast/toast/toast.tsx

```diff
@@ -55,7 +55,7 @@ export class Toast {
     const toast = toastManager.getToast();
 
     return (
-      <Host>
+      <Host role="status" aria-live="polite">
```

Comment:
is it correct, that aria-live is always defined?

Reply:
Needs to be on pre-rendered host to get announced in screen reader.